fdbclient/NativeAPI.actor.h

```diff
@@ -126,7 +126,7 @@ class Cluster : public ReferenceCounted<Cluster>, NonCopyable {
 	Future<Void> onConnected();
 
 private: 
-	void init(Reference<ClusterConnectionFile> connFile, bool startClientInfoMonitor, int apiVersion=Database::API_VERSION_LATEST);
+	void init(Reference<ClusterConnectionFile> connFile, bool startClientInfoMonitor, Reference<AsyncVar<int>> connectedCoornidatorsNum, int apiVersion=Database::API_VERSION_LATEST);
```

Comment:
I think this should be `connectedCoordinatorsNum` rather than `connectedCoornidatorsNum`.

Reply:
Thanks! I will change it soon